packages/migrate/src/__tests__/DbExecute.test.ts

```diff
@@ -635,34 +657,43 @@ COMMIT;`,
       fs.writeFileSync('script.sql', 'ThisisnotSQLitshouldfail')
       const result = DbExecute.new().parse(['--schema=./prisma/schema.prisma', '--file=./script.sql'])
       await expect(result).rejects.toThrowErrorMatchingInlineSnapshot(`
-              db error: ERROR: at or near "thisisnotsqlitshouldfail": syntax error
-              DETAIL: source SQL:
-              ThisisnotSQLitshouldfail
-              ^
+        db error: ERROR: at or near "thisisnotsqlitshouldfail": syntax error
+        DETAIL: source SQL:
+        ThisisnotSQLitshouldfail
+        ^
 
 
-            `)
+      `)
```

Comment:
(At some point we should understand how to tell jest to not add new spaces when they're useless)

Reply:
Here it's fewer spaces, which is nice and rare 😅